en/sensor_bus/i2c.md

```diff
@@ -16,7 +16,7 @@ The bus is not fast enough even with a single device attached to allow vibration
 
 Drivers should `#include <drivers/device/i2c.h>` and then provide an implementation of the abstract base class `I2C` defined in **I2C.hpp** for the target hardware (i.e. for NuttX [here](https://github.com/PX4/PX4-Autopilot/blob/master/src/lib/drivers/device/nuttx/I2C.hpp)).
 
-Drivers will also need to include headers for their type of device (**drv_*.h**) in [/src/drivers/](https://github.com/PX4/PX4-Autopilot/tree/master/src/drivers) - e.g. [drv_baro.h](https://github.com/PX4/PX4-Autopilot/blob/master/src/drivers/drv_baro.h).
+Drivers will also need to include headers for their type of device (**drv_*.h**) in [/src/drivers/](https://github.com/PX4/PX4-Autopilot/tree/master/src/drivers) - e.g. [drv_baro.h](https://github.com/PX4/PX4-Autopilot/blob/master/src/drivers/drv_baro.h). 
```

Comment:
@bkueng Is this statement true? Specifically I have highlighted this because the link to https://github.com/PX4/PX4-Autopilot/blob/master/src/drivers/drv_baro.h does not exist anymore - so perhaps this is not generally needed, or perhaps this is not needed for barometers.

Maybe instead this needs to say "use a header here IFF it exists for the type of thing you are creating?

Reply:
It's generally needed less and less. Only very few still need use it, e.g. airspeed: https://github.com/PX4/PX4-Autopilot/blob/master/src/drivers/drv_airspeed.h